apps/api/src/coscrad-cli/data-restore.cli-command.e2e.spec.ts

```diff
@@ -126,12 +126,20 @@ describe(`CLI Command: **data-restore**`, () => {
                     new ArangoQueryRunner(databaseProvider)
                 );
 
-                const databaseSnapshot = await dataExporter.fetchSnapshot();
+                const databaseSnapshot = await dataExporter.fetchSnapshot([
+                    ...Object.values(ArangoEdgeCollectionId),
+                ]);
+
+                const documentCollections = [
+                    ...Object.values(ArangoDocumentCollectionId),
+                    'games',
+                    'term__VIEWS',
+                    'vocabularyList__VIEWS',
```

Comment:
I guess this will include `photograph__VIEWS` soon?

Reply:
@yaanahuu2 Yea, for now. We're close to the point where we will dynamically register the collection names. The important point of this command is that even if you don't register the collection name, it will still be exported and imported via the dump \ restore command. That said, `arangodump` is the first line of defense. This is just a second option.